packages/react/src/components/CardCodeEditor/CardCodeEditor.jsx

```diff
@@ -74,6 +74,8 @@ const CardCodeEditor = ({
   const [isExpanded, setIsExpanded] = useState();
   const [error, setError] = useState(false);
 
+  const mergedI18n = useMemo(() => ({ ...defaultProps.i18n, ...i18n }), [i18n]);
```

Comment:
added b/c of errors thrown in testing when a single i18n key was replaced. Since there was no merge the other items were passed as undefined.